corehq/ex-submodules/casexml/apps/phone/data_providers/case/clean_owners.py

```diff
@@ -106,8 +108,8 @@ def get_payload(self):
         )
         self.restore_state.current_sync_log.log_format = LOG_FORMAT_SIMPLIFIED
         index_tree = IndexTree(indices=all_indices)
-        case_ids_on_phone = all_syncing
-        primary_cases_syncing = all_syncing - all_dependencies_syncing
+        case_ids_on_phone = all_synced
```

Comment:
this is the fix - don't use everything that was checked, use everything that actually synced as the base set